modules/machineset-yaml-azure.adoc

```diff
@@ -71,19 +110,46 @@ spec:
           resourceGroup: <infrastructureID>-rg <1>
           sshPrivateKey: ""
           sshPublicKey: ""
+ifndef::infra[]
           subnet: <infrastructureID>-<role>-subnet <1> <2>
           userDataSecret:
             name: <role>-user-data <2>
+endif::infra[]
+ifdef::infra[]
+          subnet: <infrastructureID>-infra-subnet <1> <2>
+          userDataSecret:
+            name: infra-user-data <2>
```

Comment:
Shouldn't we not be documenting using custom user-data secrets? I think they should just be using worker-user-data here right since we don't support booting into custom pools (and we can remove the if). Could you check me on that assumption

Reply:
@yuqi-zhang I am taking this PR over from Cody, in his absence. The `worker-user-data` is a default manifest?  You are suggesting we change this to:

```
          userDataSecret:
              name: worker-user-data <2>

```

Reply:
I suggest just removing the whole if condition, and modifying the original  from `<role>-user-data` to `worker-user-data`

Reply:
@yuqi-zhang PTAL: https://github.com/openshift/openshift-docs/pull/33057/commits/9dfd506453269d4f8525e828463f50bbb14f9a9c